tsconfig.json

```diff
@@ -5,6 +5,7 @@
 		"module": "commonjs",
 		"moduleResolution": "node",
 		"declaration": true,
+		"skipLibCheck": true,
```

Comment:
If we don't do this, the following error appears when compiling.

<img width="949" alt="screen shot 2018-05-02 at 22 16 44" src="https://user-images.githubusercontent.com/1913805/39546997-86db2ee0-4e56-11e8-9692-39fe86b5f961.png">